src/core/ht.h

```diff
@@ -34,28 +34,24 @@ typedef struct {
     int8_t rollInverse;
     int8_t panInverse;
 
-    float   tiltFactor; //Gain
-    float   rollFactor;
-    float   panFactor;
+    float tiltFactor; // Gain
+    float rollFactor;
+    float panFactor;
+
+    // PPM setting
+    int16_t htChannels[3]; // 0=Pan, 1=tilt, 2=roll
 
-    //PPM setting
-    int16_t htChannels[3]; //0=Pan, 1=tilt, 2=roll
-    
     // internal state
     uint8_t enable;
-	
-} ht_data_t;
 
+} ht_data_t;
 
 void init_ht();
 void enable_ht();
 void disable_ht();
 void calibrate_ht();
 void set_maxangle_ht(int angle);
```

Comment:
_personally_ i prefer naming functions `<subject>_<action>` eg, `ht_init` vs `init_ht`
this seems also be the case in a good portion of the codebase. 
can we adjust these function names?

Reply:
Agreed, and done. This also makes it more inline with all the other code.